### DISCUSSION TITLE
Aqara roller shade companion E1 (Xiaomi ZNJLBL01LM) state reporting

### DISCUSSION THREAD

**Discussion by Xitee1** [Koenkk/zigbee2mqtt#16910] [Devices]
I've got the Aqara roller shade companion E1 and paired it with z2m.
While I was able to control it, the device didn't report it's state after opening/closing. Because of this, in HA after closing it, it would still gray out the up button because it thinks it's still open. The battery state also didn't update. Manually refreshing the state in z2m worked.
I have updated everything, the device firmware, the coordinator firmware and z2m and of course re-paired the device. Still didn't work. I was already giving up and thought if I'm already here, I change the ZigBee Channel (wanted to do this for a while because I was still using the default channel 11) to 25. Paired all devices again.
By the way, I'm using the RaspBee 2.

After all that I finally closed the shutters because it was late and... it worked?!
Now it is reporting the state instantly!
So.. it seems like it was the ZigBee channel. I had to change it from 11 to 25.

Maybe this is worth mentioning in the docs?

**Comment by z-master42** [comment]
Is there anything new here? I can't imagine that a channel change would be the solution, that doesn't make sense.

**Reply by Xitee1** [reply]
No, there isn't anything new. The device to this day works without any problems after changing the channel.

**Comment by SirBogner** [comment]
I too had the same problem with my roller blind drivers and ended up switching from zigbee channel 11 to 25.
It now works as it should but I have no idea why it does not work on channel 11.
